src/lib/github.ts

```diff
@@ -0,0 +1,44 @@
+import * as core from "@actions/core";
+import * as github from "@actions/github";
+import {} from "@actions/github/lib/github";
+import { GitHub } from "@actions/github/lib/utils";
+
+let client: InstanceType<typeof GitHub>;
+function getClient(): InstanceType<typeof GitHub> {
+  if (client) {
+    return client;
+  } else {
+    const GITHUB_TOKEN = core.getInput("github-token");
+    return github.getOctokit(GITHUB_TOKEN);
+  }
+}
+
+type ActionPayload = {
```

Comment:
This is a very simple type and It will probably change when the code will evolve regarding the [deny / allow strategies task]()https://linear.app/fewlines/issue/TRB-68/add-dependancies-deny-or-allow-strategy.